livekit-rtc/livekit/rtc/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.19.1"
+__version__ = "0.19.2"
```

Comment:
If we're planning to release everything in main then we should do a minor version increase instead to 0.20.0 since this release includes data streams.